crane-cloud/frontend
# chore: include missing functionality for adding a custom domain and revert functionality under app settings

# Description

- This PR aims to include the missing pieces of the custom domain and revert functionality under an application's settings and now should be able to add/update the custom domain name or revert to the crane cloud default url.

## Type of change


## Trello Ticket ID

https://trello.com/c/F5jOcOwC

## How Can This Been Tested?

- Simply run this PR locally and visit any of your application's settings and try adding a custom domain and then reverting to the default url.
- You can also check edge cases when for example an invalid url is provided, trying to revert when an application has no existing custom domain etc.

## Checklist:


# Screenshots

<img width="1002" alt="image" src="https://github.com/crane-cloud/frontend/assets/63339234/d007d932-6e41-405c-b970-f224dadd7de0">
